strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/storage/metadata/nuget/rss/PackageFeedTest.java

```diff
@@ -31,48 +28,15 @@
  * Test class RSS
  *
  * @author sviridov
+ * @author Pablo Tirado
  */
-@ContextConfiguration(classes = { NugetBootersTestConfig.class })
```

Comment:
@ptirador can you please check doe's `NugetBootersTestConfig` used? maybe we should drop it...